From 625ff7a8cb5a025f2d7e82335363c3339d014955 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 22 Jul 2020 18:03:52 +0200
Subject: [PATCH 1455/1852] xilinx: Fix xlnx,mio_bank property

commit d1cbd86af7b106c3811e56576466716dcdd4c723 from
https://github.com/Xilinx/linux-xlnx.git

s/xlnx,mio_bank/xlnx,mio-bank/g

DT binding is describing mio-bank not mio_bank that's why fix all DTSes and
also driver itself.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm015-dc1.dts | 4 ++--
 arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm019-dc5.dts | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts      | 4 ++--
 arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts      | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revA.dts      | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revC.dts      | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu106-revA.dts      | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu111-revA.dts      | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu1275-revB.dts     | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu1285-revA.dts     | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu208-revA.dts      | 2 +-
 arch/arm64/boot/dts/xilinx/zynqmp-zcu216-revA.dts      | 2 +-
 12 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm015-dc1.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm015-dc1.dts
index 75788e0d4574..fdfb37bdaa16 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm015-dc1.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm015-dc1.dts
@@ -374,7 +374,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdhci0_default>;
 	bus-width = <8>;
-	xlnx,mio_bank = <0>;
+	xlnx,mio-bank = <0>;
 };
 
 /* SD1 with level shifter */
@@ -386,7 +386,7 @@
 	 * This property should be removed for supporting UHS mode
 	 */
 	no-1-8-v;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 };
 
 &serdes {
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm019-dc5.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm019-dc5.dts
index 9427bd57aa1b..e2d82aca6f26 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm019-dc5.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zc1751-xm019-dc5.dts
@@ -408,7 +408,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdhci0_default>;
 	no-1-8-v;
-	xlnx,mio_bank = <0>;
+	xlnx,mio-bank = <0>;
 };
 
 &ttc0 {
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
index e55871465f5f..5a9c4e4a608a 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
@@ -482,7 +482,7 @@
 	disable-wp;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdhci0_default>;
-	xlnx,mio_bank = <0>;
+	xlnx,mio-bank = <0>;
 };
 
 &sdhci1 {
@@ -490,7 +490,7 @@
 	bus-width = <0x4>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdhci1_default>;
-	xlnx,mio_bank = <0>;
+	xlnx,mio-bank = <0>;
 	non-removable;
 	disable-wp;
 	cap-power-off-card;
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
index 2bfd552a1bfa..c4bc219de6c2 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
@@ -942,7 +942,7 @@
 	no-1-8-v;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdhci1_default>;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 };
 
 &serdes {
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revA.dts
index edb50e1ac7c2..b4c5f05936a9 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revA.dts
@@ -459,7 +459,7 @@
 	no-1-8-v;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdhci1_default>;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 	disable-wp;
 };
 
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revC.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revC.dts
index c974ee80a0cc..32083b64a117 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revC.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu104-revC.dts
@@ -472,7 +472,7 @@
 	no-1-8-v;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdhci1_default>;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 	disable-wp;
 };
 
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu106-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu106-revA.dts
index 37f1ba3bf926..a9e687a8d2c5 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu106-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu106-revA.dts
@@ -939,7 +939,7 @@
 	 * This property should be removed for supporting UHS mode
 	 */
 	no-1-8-v;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 };
 
 &serdes {
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu111-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu111-revA.dts
index e727268aadfa..0973f7888da3 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu111-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu111-revA.dts
@@ -789,7 +789,7 @@
 	 * This property should be removed for supporting UHS mode
 	 */
 	no-1-8-v;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 };
 
 &serdes {
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu1275-revB.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu1275-revB.dts
index b33cbdae9fb7..e39f225540d8 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu1275-revB.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu1275-revB.dts
@@ -117,5 +117,5 @@
 	 * removed for supporting UHS mode
 	 */
 	no-1-8-v;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 };
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu1285-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu1285-revA.dts
index db85950611ae..21d62e993a33 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu1285-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu1285-revA.dts
@@ -268,5 +268,5 @@
 	 * This property should be removed for supporting UHS mode
 	 */
 	no-1-8-v;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 };
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu208-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu208-revA.dts
index 1cb8afc81285..03bc7ca2a350 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu208-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu208-revA.dts
@@ -633,7 +633,7 @@
 	 * This property should be removed for supporting UHS mode
 	 */
 	no-1-8-v;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 };
 
 &serdes {
diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu216-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu216-revA.dts
index 045cf715b708..c059e144c0f2 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu216-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu216-revA.dts
@@ -637,7 +637,7 @@
 	 * This property should be removed for supporting UHS mode
 	 */
 	no-1-8-v;
-	xlnx,mio_bank = <1>;
+	xlnx,mio-bank = <1>;
 };
 
 &serdes {
-- 
2.31.1

